zcash_primitives/src/transaction/txid.rs

```diff
@@ -536,9 +538,13 @@ impl TransactionDigest<Authorized> for BlockTxCommitmentDigester {
         orchard_bundle.map_or_else(
             orchard_bundle::commitments::hash_bundle_auth_empty,
             |b| match b {
-                OrchardVanilla(bundle) => bundle.authorizing_commitment().0,
+                OrchardVanilla(bundle) => bundle.authorizing_commitment(&BTreeMap::new()).0,
```

Comment:
can we use `&ORCHARD_SIGHASH_VERSION_TO_BYTES` here instead `new()`?

Reply:
Done